src/test/java/seedu/address/logic/commands/AddPatientCommandTest.java

```diff
@@ -0,0 +1,331 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalPatients.PATIENT_ALICE;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.function.Predicate;
+
+import org.junit.jupiter.api.Test;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.GuiSettings;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.appointments.Appointment;
+import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.dentist.Dentist;
+import seedu.address.model.person.patients.Patient;
+import seedu.address.model.treatment.Treatment;
+import seedu.address.testutil.PatientBuilder;
+
+public class AddPatientCommandTest {
+
+    @Test
+    public void constructor_nullPatient_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new AddPatientCommand(null));
+    }
+
+    @Test
+    public void execute_dentistAcceptedByModel_addSuccessful() throws Exception {
```

Comment:
probably have to rename this to patient. But other than that LGTM

Reply:
Resolved in latest commit 👍